### PR TITLE
refactor: remove redundant code

### PR DESCRIPTION
**Pull request by florian-hoenicke**

### CODE REVIEW

File: jina/peapods/pods/k8s.py [hunk 1]
```diff
@@ -304,3 +235,14 @@ def _get_base_executor_version(self):
             return jina.__version__
         else:
             return 'master'
+
+    def _create_node(self, suffix):
+        name = f'{self.name}_{suffix}'
```
Comment (slettner) [inline jina/peapods/pods/k8s.py:240]: The underscore is part of the suffix I think

**Review by Midnight-Conqueror — DISMISSED** [review]

File: jina/peapods/pods/k8s.py (hunk 1 above)
Comment (florian-hoenicke) [reply jina/peapods/pods/k8s.py]: thanks - fixed

File: jina/peapods/pods/k8s.py [hunk 2]
```diff
@@ -242,61 +213,21 @@ def deployments(self) -> List[Dict]:
...
-                        'head_host': f'{name}.{self.args.k8s_namespace}.svc.cluster.local',
-                        'head_port_in': 8081,
-                        'tail_port_out': 8082,
-                        'head_zmq_identity': self.head_zmq_identity,
-                    }
-                )
+                res.append(res.append(self._create_node('tail')))
         return res
 
     def _get_base_executor_version(self):
```
[the remark this answers is not in the document]
Comment (florian-hoenicke) [reply jina/peapods/pods/k8s.py:232]: Thanks, I see that it looks like property would be working here. However, it is a call to get the latest version. I will change the name to make it clear.

**Review by numb3r3 — APPROVED** [review]
LGTM

**Comment by hanxiao** [comment]
image
okay i see this flow tests is too big, let's breakup this folder into many folders

**Pull request merged**